### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by fradino** [G-U-N/Gen-L-Video#13]
Hello, I faced this warning when running one-shot-turning.py. Is it normal?

**Comment by G-U-N** [comment]
From my experience this is normal, just ignore it.

**Comment by fradino** [comment]
But the model of different epoch seems to generate results, the gradient is not spread.
